test/ExecutionMode_SPIR_to_SPIRV.ll

```diff
@@ -19,34 +19,24 @@
 
 ; CHECK-LLVM: [[WG]] = !{i32 128, i32 10, i32 1}
 
-target datalayout = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
-target triple = "spir-unknown-unknown"
+target datalayout = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir"
```

Comment:
Why has the target triple changed here? It doesn't seem related to your change

Reply:
I just regenerated the ir with the latest clang and specified `-triple spir` instead of `-triple spir-unknown-unknown`.
Now I have returned the  `-unknown-unknown` back